### DISCUSSION TITLE
Annotating a Quarto document

### DISCUSSION THREAD

**Discussion by sbrisard** [quarto-dev/quarto-cli#5057] [Q&A] [answered]
Hi,
Is there an equivalent to the [LaTeX pdfcomment package](https://www.ctan.org/pkg/pdfcomment) in Quarto?
Thanks!

**Comment by mcanouil** [comment, marked as answer]
If you are using `format: pdf` you can use `pdfcomment`.
For other format such as HTML-based formats, there is for example the Quarto Nutshell extension, see <https://github.com/schochastics/quarto-nutshell>.
![](https://github.com/schochastics/quarto-nutshell/raw/main/nutshell.gif)

You can also use the margin to annotate, see <https://quarto.org/docs/blog/posts/2022-02-17-advanced-layout/index.html>.

**Reply by sbrisard** [reply]
Thanks, I will look into nutshell.
S
